system/jlib/jlog.cpp

```diff
@@ -2379,7 +2379,8 @@ void setupContainerizedLogMsgHandler()
             if (!isEmptyString(logAudiences))
                 msgAudiences = logMsgAudsFromAbbrevs(logAudiences);
 
-            Owned<ILogMsgFilter> filter = getCategoryLogMsgFilter(msgAudiences, msgClasses, logDetail);
+            const bool local = true; // Do not include remote messages from other components
```

Comment:
Does this stop ESP from sending the messages, or just stop Dali from reporting them?

Reply:
I am not sure if this is related. The existing ESP does access dali's audit log (ex. LOG(MCauditInfo,",FileAccess,EspProcess,READ,%s,%s,%s,jobid=%s,expirySecs=%d", cluster.str(), userID.str(), fileName.str(), requestId, expirySecs);) @rpastrana @jakesmith

Reply:
I think this stops dali from receiving these messages.  The filter for messages that should be sent to dali is deduced from the filters on the logging.
Dali has a separate audit filter (set up in daserver.cpp~630).  Because that filter is not local it will be sent to anything that connects - which will ensure the audit messages are serialized.